lib/python/util.py

```diff
@@ -134,3 +134,46 @@ def pass_netcdf_checker(netcdf_file_path, tests=['cf:latest', 'imos:latest']):
     if all(return_values):
         return True # all tests passed
     return False # at least one did not pass
+
+def download_list_urls(list_url, logger):
```

Comment:
logger should be explained in this header as it is now out of context. Maybe logger should be removed from here or handled differently